source/style/code-highlight.css

```diff
@@ -0,0 +1,173 @@
+/* PrismJS 1.9.0
+http://prismjs.com/download.html?themes=prism-okaidia&languages=markup+css+clike+javascript+bash+go+graphql+java+json+php+powershell+pug+python+jsx+scss+scala */
```

Comment:
Can you also add:

- Diff
- Ruby
- Rust
- Swift
- TypeScript

These also need to be added to `supportedLang`. You also need to add the missing languages there, like Java.